documentation/vMapping_tutorial.md

```diff
@@ -0,0 +1,67 @@
+#vMapping App
```

Comment:
this app is probably redundant and we want to make sure undistortion can happen with the new module - tested on the robot!